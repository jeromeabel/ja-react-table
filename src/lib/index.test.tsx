import renderer from 'react-test-renderer';
import { describe, expect, test } from 'vitest';
import HelloWorld from './index';

describe('HelloWorld', () => {
  test('HelloWorld component renders correctly', () => {
    const component = renderer.create(<HelloWorld />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('The greetee prop works', () => {
    const component = renderer.create(<HelloWorld greetee={'Universel'} />);
    const tree = component.toJSON();
    console.log('TREE', tree);
    expect(tree).toMatchSnapshot();
  });
});
