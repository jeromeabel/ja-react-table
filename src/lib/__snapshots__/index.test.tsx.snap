// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HelloWorld > HelloWorld component renders correctly 1`] = `
<div
  className="bg-green-500 text-2xl"
>
  Hello, 
  World
  !
</div>
`;

exports[`HelloWorld > The greetee prop works 1`] = `
<div
  className="bg-green-500 text-2xl"
>
  Hello, 
  Universe
  !
</div>
`;
